src/main/java/seedu/duke/commands/CommandHandler.java

```diff
@@ -1,6 +1,11 @@
 package seedu.duke.commands;
 
 import seedu.duke.exceptions.DukeError;
+import seedu.duke.exceptions.OngoingExGenerationError;
+import seedu.duke.exceptions.OngoingExHelpError;
+import seedu.duke.exceptions.OngoingExProgressError;
+import seedu.duke.exceptions.OngoingExWriteError;
+import seedu.duke.exceptions.OngoingExHistoryError;
```

Comment:
Good attempt in adding the exceptions!
Seems like they are under the same "category" ?
@Others
```suggestion
import seedu.duke.exceptions.OngoingExerciseError;
```